docs/api-docs/catalog/v2-vs-v3.md

```diff
@@ -102,7 +102,7 @@ In V3, any variants or modifier adjusters created with non-null core properties
 * [Deprecations and sunsets](/api-docs/getting-started/deprecations-and-sunsets)
 
 ### Endpoints
-* [V3 Product Complex Rules](/api-reference/store-management/catalog/product-complex-rules)
-* [V3 Product Options](/api-reference/store-management/catalog/product-options)
-* [V3 Product Modifiers](/api-reference/store-management/catalog/product-modifiers)
-* [V3 Product Variants](/api-reference/store-management/catalog/product-variants)
+* [V3 Product Complex Rules](/api-reference/store-management/catalog/product-complex-rules/getcomplexrules)
+* [V3 Product Options](/api-reference/store-management/catalog/product-variant-options/getoptions)
```

Comment:
Per your decision on the preceding comment, something like one of the following:
```suggestion
* [V3 Product Variant Options](/api-reference/store-management/catalog/product-variant-options/getoptions)
```
or 
```suggestion
* [V3 Products: Get product variant options](/api-reference/store-management/catalog/product-variant-options/getoptions)
```